src/main/resources/application.yml

```diff
@@ -3,12 +3,11 @@ server:
 
 auth:
   jwt:
-    publicKeyUrl: "http://localhost:8081/oauth/token/public_key"
+    publicKeyUrl: "https://ego.dev.argo.cancercollaboratory.org/api/oauth/token/public_key"
```

Comment:
please change this back to what it was. we dont want to expose how to reach our internal services